pkg/cloudprovider/gce/gce.go

```diff
@@ -444,6 +490,19 @@ func (gce *GCECloud) EnsureTCPLoadBalancerDeleted(name, region string) error {
 			return err
 		}
 	}
+	fwName := makeFirewallName(name)
```

Comment:
You don't have to change this, but why did this go between forwarding rules and target pools? Reversing the order from create seems most natural.